docs/section22/Rule22-14.md

```diff
@@ -17,15 +17,21 @@
 **Data Lookup:** None  
 **Function Call:**  
 
-1. get_baseline_system_types()
+1. get_heat_rejection_loops_connected_to_baseline_systems()
 
 **Applicability Checks:**  
 
-- Get B-RMR system types: `baseline_hvac_system_dict = get_baseline_system_types(B-RMR)`
+- The function get_heat_rejection_loops_connected_to_baseline_systems() returns a list of loops that are connected to Type 7,8,12,13,7b,8b,12b.  Get the list of applicable heat rejection loops: `heat_rejection_loop_ids = get_heat_rejection_loops_connected_to_baseline_systems(B_RMI)`
 
-  - Check if B-RMR is modeled with at least one air-side system that is Type-7, 8, 11.1, 11.2, 12, 13, 7b, 8b, 11b, 12b, 13b, i.e. with air-side system served by chiller(s), continue to the next applicability check: `if any(sys_type in baseline_hvac_system_dict.keys() for sys_type in ["SYS-7", "SYS-8", "SYS-11.1", "SYS-11.2", "SYS-12", "SYS-13", "SYS-7B", "SYS-8B", "SYS-11B", "SYS-12B"]): CHECK_RULE_LOGIC`
+- check applicability for each HeatRejection in the model: `for heat_rejection in B_RMI.heat_rejections:`
 
-  - Else, rule is not applicable to B-RMR: `else: RULE_NOT_APPLICABLE`
+  - check if heat_rejection fluid_loop is one of the applicable loops: `if heat_rejection.loop.id in heat_rejection_loop_ids:`
+  
+    - check if the design wetbulb temperature is between 55 and 90F (inclusive): `if ( heat_rejection.design_wetbulb_temperature >= 55 ) AND ( heat_rejection.design_wetbulb_temperature <= 90 ): CHECK_RULE_LOGIC`
+
+    - Else, rule is not applicable to the heat rejection: `else: RULE_NOT_APPLICABLE`
+
+  - Else, rule is not applicable to the heat rejection: `else: RULE_NOT_APPLICABLE`
```

Comment:
Same comment as on 22-13. It seems that the RDS should account for cases with multiple heat rejection loops and produce Pass outcome only if design temperature rise is correct for all of them.

Reply:
this rule is written to provide an outcome for each heat rejection loop.